src/dashboard/index.js

```diff
@@ -37,6 +38,230 @@ function render() {
+                search_button.setAttribute('class', 'btn btn-danger')
+                search_button.setAttribute('type', 'button')
+                var text = document.createTextNode("search");
+                search_button.appendChild(text)
+                input_grp.appendChild(input_text)
+                input_grp.appendChild(search_button)
+
+                custom_search_input.appendChild(input_grp)
+                search_bar_block.appendChild(custom_search_input)
+                md_12_class.appendChild(search_bar_block)
+                row_class.appendChild(md_12_class)
+                container_class.appendChild(row_class)
+                activity_folder.appendChild(container_class)
+                activity_folder.appendChild(clear_fix)
+
+                // tables creation
+                var div_table_responsive = document.createElement('div'); // div to encapsulate table
+                div_table_responsive.setAttribute("class", "table-responsive");
+
+                var table = document.createElement('table');
+                table.setAttribute("id", "mytable");
+                table.setAttribute("class", "table table-bordred table-striped");
+
+                var thead =  document.createElement('thead');
+                var tboady = document.createElement('tbody');
+
+
+                // var tr = document.createElement('tr');
+                var arrheader = [' ', 'Cookbooks', 'Repository', 'Recipes', 'Policyfile', 'Created at', 'Action'];
+
+
+                // this has to come dynamically from reading directory structure (This should come from api)
+               //  todo - Api will fetch this data
+               var ele = {
+                   check_list: '',
+                   cookbooks: 'Zen_apache',
+                   repository: 'my_repo',
+                   recipes: '20(show >>)',
+                   policyfile: 'config.rb',
+                   created_at: '0 am, 27/3/22',
+                   action: 'Upload'
+               }
+
+               var array =  Array(50).fill(ele)
+
+                for (var j = 0; j < arrheader.length; j++) {
+                    var th = document.createElement('th'); //column
+                    var text = document.createTextNode(arrheader[j]); //cell
+                    th.appendChild(text);
+                    thead.appendChild(th);
+                }
+                table.appendChild(thead);
+
+                for (var i = 0; i < array.length; i++) {
+                    var tr = document.createElement('tr');
+
+                    var td1 = document.createElement('td');
+
+                    var input_1 =  document.createElement('input'); //input form
+                    input_1.setAttribute("type", "checkbox"); // this will be set based on input
+                    input_1.setAttribute("class", "checkthis");
+
+                    // check if it is is checked
+
+                    var td2 = document.createElement('td');
```

Comment:
This could be handled dynamically

Reply:
We can definitely optimized it through the loop, I had given it a thought, 
e.g
 ```
for (var i = 1; i <7; i++) {
   var tdi =  document.createElement('td');
}
```

it was having some issue, will try and optimize it again.